README.md

```diff
@@ -111,14 +111,25 @@ aws sts get-session-token  --endpoint-url http://localhost:12345 --region localh
 
 STS allows NPA (non personal account) access, in cases where client is not able to authenticate
 with Keycloak server. 
-In order to notify STS that user is NPA user, manual insert to db (users table) is needed.
+In order to notify STS that user is NPA user, below steps needs to be done:
 
-Either create sql script, or run insert in STS database (mariadb)
+1. User needs to be in administrator groups (user groups are taken from Keycloak)
+
+2. Check settings of the value `STS_ADMIN_GROUPS` in application.conf and set groups accordingly. Config accepts 
+array in format of ["testgroup", "othergroup"]
```

Comment:
The documentation is a bit misleading IMHO, I was getting 500 because I was using `["testgroup", "othergroup"]` instead of `"testgroup,othergroup"`.